### PR TITLE
Updates to Update EOP License File article

### PR DESCRIPTION
**Pull request by terra4383**
Please verify second step of first section and the last section in general - do we want to say this? Thanks.

### CODE REVIEW

**Comment by SteveFeldman** [comment]
OK...looks good.

File: content/admin/teamserver/faq/HowToUpdateEOPLicenseFile.md [hunk 1]
```diff
@@ -4,43 +4,46 @@ description: "Overview of the two license update options"
...
-Contrast Enterprise-On-Premise (EOP) customers may occasionally need a new license file. There are two possible approaches for updating this file. The license can be updated by logging into the application using the SuperAdmin account.  Alternatively, the license file can be replaced on the local file system.
+Contrast Enterprise-On-Premise (EOP) customers may occasionally need a new license file. There are two possible approaches for updating this file. The license can be updated in the UI by logging into the application as a SuperAdmin.  Alternatively, the license file can be replaced on the local file system.
 
 If the license is expired, then the second method (file system) below must be used.
 
 ## Updating Your License File Through The UI
 
-1. Obtain a new license from [Hub](https://hub.contrastsecurity.com/h/), your account manager, or the technical support team
-2. Login to TeamServer with the SuperAdmin account:
+1. Obtain a new license from [Hub](https://hub.contrastsecurity.com/h/), your account manager, or the technical support team.
```
Comment (cgtwin) [inline content/admin/teamserver/faq/HowToUpdateEOPLicenseFile.md]: I think this step is not required or at least it's not clear what should I do

Comment (terra4383) [inline content/admin/teamserver/faq/HowToUpdateEOPLicenseFile.md]: @cgtwin Are you referring to step 1?

Comment (cgtwin) [inline content/admin/teamserver/faq/HowToUpdateEOPLicenseFile.md]: yes, step #1

**Pull request merged**